File: PYTHON/Kode Program/Harddrive.py
from Komponen import Komponen

class Harddrive(Komponen):
    def __init__(self, kapasitas=0, tipe="", merk="", harga=0):
        super().__init__(merk, harga)
        self._kapasitas = kapasitas
        self._tipe = tipe

    def set_kapasitas(self, kapasitas):
        self._kapasitas = kapasitas

    def get_kapasitas(self):
        return self._kapasitas

    def set_tipe(self, tipe):
        self._tipe = tipe

    def get_tipe(self):
        return self._tipe

File: JAVA/Kode Program/Harddrive.java
public class Harddrive extends Komponen {
    private int kapasitas;
    private String tipe;

    public Harddrive(int kapasitas, String tipe, String merk, int harga) {
        super(merk, harga);
        this.kapasitas = kapasitas;
        this.tipe = tipe;
    }

    public void setKapasitas(int kapasitas) {
        this.kapasitas = kapasitas;
    }

    public int getKapasitas() {
        return this.kapasitas;
    }

    public void setTipe(String tipe) {
        this.tipe = tipe;
    }

    public String getTipe() {
        return this.tipe;
    }
}
